async.go

```diff
@@ -68,7 +68,7 @@ func processAsyncResults() {
 
 		// push to Core Data
 		event := &models.Event{Device: acv.DeviceName, Readings: readings}
-		_, err := common.EventClient.Add(event)
+		_, err := common.EventClient.Add(event, nil)
```

Comment:
If I'm not wrong, instead of `nil` should be `context.Background()`. @tsconn23 can clarify trhis